src/bin.ts

```diff
@@ -210,6 +210,10 @@ const deploy = async ({ yes, bucket, userAgent }: { yes: boolean; bucket: string
         spinner.text = 'Listing objects...';
         spinner.color = 'green';
         const objects = await listAllObjects(s3, config.bucketName, config.bucketPrefix);
+        const keyTagMapObjects = objects.reduce((acc: any, curr) => {
```

Comment:
I think `keyToETagMap` would be a more clear name for this variable. I'd also prefer `result` over `acc`. In my mind accumulators are numeric.

Reply:
For the result over acc. i'll need to wrap that line with /* eslint-disable no-param-reassign */ because it gets triggered :/ the only variable name not getting it trigger is acc and accumulator. 

do you feel strongly about that? i feel weird wrap those lines with a disable over a variable name lol

Reply:
Ah, I wasn't expecting that. It's a tricky one, it does feel weird to have to disable linting for it.

Disabling linting is what's suggested [here](https://stackoverflow.com/a/57331012/674675) and [here](https://github.com/eslint/eslint/issues/8581#issuecomment-348216427). But in searching around this issue I have found that `acc` seems to be used as an identifier commonly for this pattern, so I'm happy for it to be left as-is.

I'm happy with this PR now, but I'm going to leave it open for a bit longer to give jariz a chance to explain how I can run tests on it.